druid-shell/src/platform/gtk/window.rs

```diff
@@ -818,6 +818,11 @@ impl WindowHandle {
         }
     }
 
+    pub fn get_content_insets(&self) -> Insets {
```

Comment:
I'd like to just call this `content_insets` here and elsewhere, `get_` is considered 'not idiomatic'.

Reply:
Ok, was just matching the other methods here